python/src/trezorlib/debuglink.py

```diff
@@ -239,11 +275,12 @@ def button_request(self, br):
             raise AssertionError("input flow ended prematurely")
         else:
             try:
+                assert isinstance(self.input_flow, Generator)
                 self.input_flow.send(br)
             except StopIteration:
                 self.input_flow = self.INPUT_FLOW_DONE
 
-    def get_pin(self, code=None):
+    def get_pin(self, code: Any = None) -> str:
```

Comment:
`code` is `Optional[PinMatrixRequestType]`

Reply:
Done in 0c94cfcb5